### PR TITLE
RDoc-1931_StatsViewSQL - SQL ETL ongoing tasks stats documentation

### PR DESCRIPTION
**Pull request by reebhub**

### CODE REVIEW

**Review by Danielle9897 — APPROVED** [review]

**Comment by Danielle9897** [comment]
Todo: Need to add both the Olap Etl & Sql Etl to the **overview list page**

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 1]
```diff
@@ -0,0 +1,143 @@
+﻿# Ongoing Tasks Stats: OLAP ETL Stats
+---
+
+{NOTE: }
+
+* **OLAP ETL** is a process that reads data from a RavenDB database, 
+  transforms it, and stores it locally or remotely in a format that 
+  OLAP (OnLine Analytical Processing) can be performed upon.
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:8]: =>
**OLAP ETL** is a process that reads data from a RavenDB database, transforms it, and stores it locally or remotely in a parquet format, ready for **OnLine Analytical Processing** (OLAP).

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+4. **Task Bar**  
+    * Hover over the bar to display a tooltip with the task's information.  
+    * Click the bar for the expanded view.  
+    * Click and Drag the bar to slide the graph.  
+    * Zoom in & out using the mouse wheel.  
+
+---
+
+### OLAP ETL Stats Expanded View
+![OLAP ETL Stats Expanded View](images/stats-view-16-olap-etl-extracted-view.png "OLAP ETL Stats Expanded View")
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:39]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+   * **Duration**  
+     The time it took to transfer the documents to their destination.  
+   * **Successfully Loaded**  
+     Documents transfer verification.  
+   * **Last Loaded Etag**  
+     Last loaded document's identifier.  
+
+* **Load/Local**  
+   * **Duration**  
+     The time it took to store **all documents** on the local storage.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:81]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+* **Load/Local/X**  
+   * **Duration**  
+     The time it took to store **each document** on the local storage.  
+       {NOTE: }
+       The document is stored locally before it is transferred to the remote destination.
+       {NOTE/}  
+
+* **Load/Upload/X**  
+   * **Duration**  
+     The time it took to upload **each document** to the remote destination.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:102]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+       {NOTE: }
+       The document is stored locally before it is transferred to the remote destination.
+       {NOTE/}  
+
+* **Load/Upload/X**  
+   * **Duration**  
+     The time it took to upload **each document** to the remote destination.  
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload/X** bar will not be displayed.
+       {NOTE/}
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:105]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+* **Load/Local**  
+   * **Duration**  
+     The time it took to store **all documents** on the local storage.  
+       {NOTE: }
+       The documents are stored locally before they are transferred to a remote destination.
+       {NOTE/}
+
+* **Load/Upload**  
+   * **Duration**  
+     The time it took to upload **all documents** to the remote destination.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:88]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+     The time it took to store **all documents** on the local storage.  
+       {NOTE: }
+       The documents are stored locally before they are transferred to a remote destination.
+       {NOTE/}
+
+* **Load/Upload**  
+   * **Duration**  
+     The time it took to upload **all documents** to the remote destination.  
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload** bar will not be displayed.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:90]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+       {NOTE/}
+
+* **Load/Local/X**  
+   * **Duration**  
+     The time it took to store **each document** on the local storage.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:95]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+      * The transfer Date and Time  
+      * The document Database Name  
+      * The OLAP ETL Task Name  
+      * The ETL Script Name  
+   * **Upload Properties**, including -  
+      * Destination Type  
+        Can be Amazon S3, Amazon Glacier, Microdoft Azure, Google Cloud Platform, or FTP.  
+        {NOTE: }
+        If the destination is the local storage, the **Upload Properties** will not be displayed.
+        {NOTE/}
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:117]: removed

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+       {NOTE/}  
+
+* **Load/Upload/X**  
+   * **Duration**  
+     The time it took to upload **each document** to the remote destination.  
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload/X** bar will not be displayed.
+       {NOTE/}
+
+   * **File Name** on the remote destination, constructed of -
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:107]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+
+* **Load/Upload/X**  
+   * **Duration**  
+     The time it took to upload **each document** to the remote destination.  
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload/X** bar will not be displayed.
+       {NOTE/}
+
+   * **File Name** on the remote destination, constructed of -  
+      * The transfer Date and Time
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:108]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+* **Load/Upload/X**  
+   * **Duration**  
+     The time it took to upload **each document** to the remote destination.  
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload/X** bar will not be displayed.
+       {NOTE/}
+
+   * **File Name** on the remote destination, constructed of -  
+      * The transfer Date and Time  
+      * The document Database Name
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:109]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+   * **Duration**  
+     The time it took to upload **each document** to the remote destination.  
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload/X** bar will not be displayed.
+       {NOTE/}
+
+   * **File Name** on the remote destination, constructed of -  
+      * The transfer Date and Time  
+      * The document Database Name  
+      * The OLAP ETL Task Name
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:110]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+     The time it took to upload **each document** to the remote destination.  
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload/X** bar will not be displayed.
+       {NOTE/}
+
+   * **File Name** on the remote destination, constructed of -  
+      * The transfer Date and Time  
+      * The document Database Name  
+      * The OLAP ETL Task Name  
+      * The ETL Script Name
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:111]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload/X** bar will not be displayed.
+       {NOTE/}
+
+   * **File Name** on the remote destination, constructed of -  
+      * The transfer Date and Time  
+      * The document Database Name  
+      * The OLAP ETL Task Name  
+      * The ETL Script Name  
+   * **Upload Properties**, including -
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:112]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload** bar will not be displayed.
+       {NOTE/}
+
+* **Load/Local/X**  
+   * **Duration**  
+     The time it took to store **each document** on the local storage.  
+       {NOTE: }
+       The document is stored locally before it is transferred to the remote destination.
+       {NOTE/}
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:98]: removed

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+
+* **Load/Upload**  
+   * **Duration**  
+     The time it took to upload **all documents** to the remote destination.  
+       {NOTE: }
+       If the destination is the local storage, the **Load/Upload** bar will not be displayed.
+       {NOTE/}
+
+* **Load/Local/X**
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:93]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:8]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+
+* **Load/Upload/X**  
+   * **Duration**  
+     The time it took to upload document X to the remote destination.  
+
+   * **Parquet File Name** on the remote destination, constructed of -  
+      * The upload date and time  
+      * The document database name  
+      * The OLAP ETL task name  
+      * The ETL script name
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:108]: Let's remove the 'The' - can do w/o - just:

  * Upload date and time  
  * Document database name  
  * OLAP ETL task name  
   * ETL script name

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown [hunk 19]
```diff
@@ -0,0 +1,137 @@
+﻿# Ongoing Tasks Stats: OLAP ETL Stats
+---
+
+{NOTE: }
+
+* **OLAP ETL** is a process that reads data from a RavenDB database, 
+  transforms it, and stores it in a parquet format, ready for OnLine 
+  Analytical Processing (OLAP).
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:8]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown (hunk 18 above)
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/stats/ongoing-tasks-stats/olap-etl-stats.markdown:108]: done

**Review by Danielle9897 — APPROVED** [review]

**Pull request merged**